src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,104 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+import { getNumbers } from '../../utils';
+
+type Props = {
+  currentPage: number,
+  itemsToDisplay: Array<string>[],
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination = ({
+  currentPage,
+  itemsToDisplay,
+  onPageChange,
+}: Props) => {
+  const qntyOfCells = getNumbers(1, itemsToDisplay.length);
+  const leftArrowDisable = currentPage === 1;
+  const rightArrowDisable = currentPage === itemsToDisplay.length;
+
+  const handleCellClick = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    const leftRightArrow = event.currentTarget.getAttribute('href');
+
+    if (leftRightArrow === '#prev') {
+      return onPageChange(currentPage - 1);
+    }
+
+    if (leftRightArrow === '#next') {
+      return onPageChange(currentPage + 1);
+    }
+
+    const cellValue = +event.currentTarget.innerText;
+
+    return cellValue !== currentPage && onPageChange(cellValue);
```

Comment:
The condition `cellValue !== currentPage` can be confusing. It's recommended to wrap it in parentheses for better readability.